cli/tools/repl/session.rs

```diff
@@ -299,8 +300,9 @@ impl ReplSession {
   }
 
   pub async fn closing(&mut self) -> Result<bool, AnyError> {
+    let expression = format!(r#"{}.closed"#, *REPL_INTERNALS_NAME);
     let closed = self
-      .evaluate_expression("(this.closed)")
+      .evaluate_expression(&expression)
```

Comment:
Won't that only set on globalThis?

Reply:
It would, but if you delete it then calling `close()` will just throw an error saying that it's not defined. Which is okay. But since we check value of `globalThis.closed` in the REPL look it causes a panic when deleting `globalThis`.

Reply:
Yeah, but this code always returns `false` now because it's only set on startup. It seems the `repl::close_command` is failing now.

Reply:
I thought this.closed is always evaluated on every access since it is a getter, but from the test I can see that it is only evaluated once. I think I will change this line "closed: this.closed" to getter so it should be evaluated every access. But when the closed property is deleted should I set a default value to return or what??

Reply:
I guess another option that we can try is to save the reference to `globalThis` in the "internals" object and then access the getter from this saved reference.

Reply:
@bartlomieju Using the getter won't prevent the panic issue.  Alternatively, should I return a default value if this.closed is returned undefined?